tests/integration/cattletest/core/test_resource_based_scheduling.py

```diff
@@ -95,6 +95,29 @@ def test_resource_based_scheduler(new_context, super_client):
                              'type': 'computePool'}])
 
 
+def test_delete_host(new_context, super_client):
```

Comment:
I didn't. Seems it will pass also. The test is not actually exercising the logic we wrote. When we deleted the sim host, the container gets deleted also. Need to think about another approach to fake the scenario we encountered.